config/initializers/dragonfly.rb

```diff
@@ -9,19 +9,19 @@
 
   url_format "/media/:job/:name"
 
-  # if Rails.env.development? || Rails.env.test?
-    # datastore :file,
-    #         root_path: Rails.root.join('public/system/dragonfly', Rails.env),
-    #         server_root: Rails.root.join('public')
+  if Rails.env.development? || Rails.env.test?
+    datastore :file,
+            root_path: Rails.root.join('public/system/dragonfly', Rails.env),
+            server_root: Rails.root.join('public')
 
-  # else
+  else
     datastore :s3,
       region: 'ap-southeast-2',
       bucket_name: 'personalmasterclass-dev',
-      access_key_id: 'AKIAJ4YR5WINZR5IMHCA',
```

Comment:
We need to used environment variables for api keys or any other sensitive data, have a look into: https://github.com/laserlemon/figaro